server/TypeDBServer.java

```diff
@@ -270,10 +270,10 @@ private static void runServer(ServerSubcommand.Server subcmdServer) {
         TypeDBServer server = new TypeDBServer(subcmdServer.config(), subcmdServer.isDebug());
         server.start();
         Instant end = Instant.now();
-        server.logger().info("- version: {}", Version.VERSION);
-        server.logger().info("- listening to address: {}:{}", server.address().getHostString(), server.address().getPort());
-        server.logger().info("- data directory configured to: {}", server.dataDir());
-        server.logger().info("- bootup completed in: {} ms", Duration.between(start, end).toMillis());
+        server.logger().info("version: {}", Version.VERSION);
+        server.logger().info("listening to address: {}:{}", server.address().getHostString(), server.address().getPort());
+        server.logger().info("data directory configured to: {}", server.dataDir());
+        server.logger().info("bootup completed in: {} ms", Duration.between(start, end).toMillis());
```

Comment:
Remove the `-` because Log4J already adds a `-`, so it used to look like this:
```
10:49:27.466 [main] INFO com.vaticle.typedb.core.server.TypeDBServer - - version: 2.8.1
```
Now it looks like this:
```
10:49:27.466 [main] INFO com.vaticle.typedb.core.server.TypeDBServer - version: 2.8.1
```